### PR TITLE
Update centrifuge/centrifuge

### PR DESCRIPTION
**Pull request by sofstam**
This PR fixes one subcommand per module (centrifuge/centrifuge)

## PR checklist

Closes [#833](https://github.com/nf-core/modules/issues/833)

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by sofstam** [comment]
I updated the `db_name`. When I untar the minigut_cf.tar.gz, I get the following files (I think it is a bit strange, if I untar the files locally, I do not get any filenames starting with `._`)

```
minigut_cf/._minigut_cf.4.cf
minigut_cf/minigut_cf.4.cf
minigut_cf/._minigut_cf.1.cf
minigut_cf/minigut_cf.1.cf
minigut_cf/._minigut_cf.2.cf
minigut_cf/minigut_cf.2.cf
minigut_cf/._minigut_cf.3.cf
minigut_cf/minigut_cf.3.cf
```

That is why I added  `-not -name "._*"` as well.

File: modules/centrifuge/centrifuge/main.nf [hunk 1]
```diff
@@ -43,8 +41,9 @@ process CENTRIFUGE {
     }
     def sam_output = sam_format ? "--out-fmt 'sam'" : ''
     """
+    db_name=`find -L ${db} -name "*.1.cf" -not -name "._*"  | sed 's/.1.cf//'`
```
Comment (jfy133) [inline modules/centrifuge/centrifuge/main.nf:44]:
```suggestion
    // we add `-name-name ._` to ensure silly Mac OSX metafiles files aren't included 
    db_name=`find -L ${db} -name "*.1.cf" -not -name "._*"  | sed 's/.1.cf//'`
```

File: modules/centrifuge/centrifuge/main.nf [hunk 2]
```diff
@@ -31,7 +29,7 @@ process CENTRIFUGE {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def paired = meta.single_end ? "-U ${reads}" :  "-1 ${reads[0]} -2 ${reads[1]}"
-    def db_name = db.toString().replace(".tar.gz","")
+    //def db_name = db.toString().replace(".tar.gz","")
```
Comment (jfy133) [inline modules/centrifuge/centrifuge/main.nf:32]:
```suggestion
```

If not being used

File: tests/modules/centrifuge/centrifuge/main.nf [hunk 3]
```diff
@@ -2,37 +2,37 @@
...
+include { CENTRIFUGE_CENTRIFUGE } from '../../../../modules/centrifuge/centrifuge/main.nf'
 
-workflow test_centrifuge_single_end {
+workflow test_centrifuge_centrifuge_single_end {
     input = [ [ id:'test', single_end:true ], // meta map
               [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
             ]
     db    =  [ [], file('https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/delete_me/minigut_cf.tar.gz', checkIfExists: true) ]
-    db_name = "minigut_cf"
+    //db_name = "minigut_cf"
```
Comment (jfy133) [inline tests/modules/centrifuge/centrifuge/main.nf:13]:
```suggestion
```

File: tests/modules/centrifuge/centrifuge/main.nf [hunk 4]
```diff
@@ -2,37 +2,37 @@
...
 
-workflow test_centrifuge_paired_end {
+workflow test_centrifuge_centrifuge_paired_end {
     input = [ [ id:'test', single_end:false ], // meta map
               [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true),
                 file(params.test_data['sarscov2']['illumina']['test_2_fastq_gz'], checkIfExists: true) ]
             ]
      db    =  [ [], file('https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/delete_me/minigut_cf.tar.gz', checkIfExists: true) ]
-     db_name = "minigut_cf"
+     //db_name = "minigut_cf"
```
Comment (jfy133) [inline tests/modules/centrifuge/centrifuge/main.nf:29]:
```suggestion
```

**Review by jfy133 — APPROVED** [review]
I think this looks good now, once we remove a bit of debugging stuff!

(Hopefully the last time we have to update this :smirk: )

File: tests/modules/centrifuge/centrifuge/main.nf (hunk 4 above)
Comment (sofstam) [reply tests/modules/centrifuge/centrifuge/main.nf:28]: I will update the PR.

File: modules/centrifuge/centrifuge/main.nf (hunk 1 above)
Comment (jfy133) [reply modules/centrifuge/centrifuge/main.nf:44]: oops yes, bash not nextflow, good catch - my bad!

**Comment by sofstam** [comment]
I updated the `meta.yml` file as well.

File: tests/modules/centrifuge/centrifuge/main.nf (hunk 4 above)
Comment (jfy133) [reply tests/modules/centrifuge/centrifuge/main.nf:28]: This is still present?

Comment (sofstam) [reply tests/modules/centrifuge/centrifuge/main.nf:28]: No, it is not.

**Pull request merged**